schemas/blocks/list_collector.json

```diff
@@ -18,6 +18,35 @@
     "required": ["id", "value"],
     "additionalProperties": false
   },
+  "summary": {
+    "type": "object",
+    "properties":{
+      "type": {
+          "type": "string",
+          "enum": ["ListCollectorSummary"]
```

Comment:
Slightly confused on why we have a type for a 'Summary' that is defined within the list collector itself?. What other types would there be?

Reply:
You're right, this isn't necessary